scripts/datasets/DET.py

```diff
@@ -0,0 +1,276 @@
+"""this script is used to prepare DET dataset for tracking,
```

Comment:
According to PEP I guess filename should be all lowercase letters, the name of the file better be ilsvrc_det.py ?

Reply:
Agree.